kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/commands/converters/impl/NumberConverter.kt

```diff
@@ -39,4 +39,4 @@ public class NumberConverter(
 
     override suspend fun toSlashOption(arg: Argument<*>): OptionsBuilder =
         IntChoiceBuilder(arg.displayName, arg.description).apply { required = true }
-}
+}
```

Comment:
You removed the newline at the end of this file for some reason — don't do that, it'll fail the build. All files must have a newline at the end.

Reply:
hmm, not sure why this happened, must of been an accident, ill fix this